### PR TITLE
Uploaded files from another repository

### PR DESCRIPTION
**Pull request by yashksaini-coder**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: "leetcode-box"
+author: puiiyuen
+description: "Update a pinned gist to show your LeetCode stats"
+branding:
+  icon: code
+  color: blue
+runs:
+  using: "node12"
```
Comment (yashksaini-coder) [inline action.yml:8]: changes the node version

**Review by yashksaini-coder — COMMENTED** [review]
All files are uploaded some name changing issues are there. 
Overall files are good

File: action.yml (hunk 1 above)
Comment (ghost) [reply action.yml:8]: Wow still using node 12 🤣

Comment (neovim-dev) [reply action.yml:8]: Transferred files from another repo and reviewed the PR before merging. I think that's a good habit for a developer @deployer117

**Pull request merged**